manifests/a/AVerMedia/RECentral/4.7.44.1/AVerMedia.RECentral.installer.yaml

```
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.2.0.schema.json
+
+PackageIdentifier: AVerMedia.RECentral
+PackageVersion: 4.7.44.1
+Installers:
+- Architecture: x64
+  InstallerType: nullsoft
+  InstallerUrl: https://storage.avermedia.com/web_release_www/software/RECentral_4_v4.7.44.1_22061501_x64.exe
+  InstallerSha256: 3A88685D8BF9CACABB365600C4E2A75362A433B9D886D63B7E01A0096E9F1CA8

```

```suggestion
  InstallerSha256: 3A88685D8BF9CACABB365600C4E2A75362A433B9D886D63B7E01A0096E9F1CA8
  ProductCode: '{ea71898b-cb0a-492a-9729-11d2c7b37e83}'
```
Product code is needed to differentiate from the installer